build.gradle

```diff
@@ -186,6 +186,7 @@ dependencies {
 
     compileOnly group: 'org.jetbrains', name: 'annotations', version:'17.0.0'
     compileOnly group: 'com.yuvimasory', name: 'orange-extensions', version: '1.3.0'
+    compileOnly group: 'com.github.spotbugs', name: 'spotbugs-annotations', version: '3.1.12'
```

Comment:
Is spotbugs required?

Reply:
Its a compile only dependencies - users won't see it.